### PR TITLE
`set variable` statement test adjustment

### PR DESCRIPTION
**Pull request by lanchengx**
Fixes https://github.com/apache/shardingsphere/issues/13460

Changes proposed in this pull request:
- Mock `ContextManager`

### CODE REVIEW

**Comment by lanchengx** [comment]
@TeslaCN could you have a  review ?

File: shardingsphere-proxy/shardingsphere-proxy-backend/src/test/java/org/apache/shardingsphere/proxy/backend/text/distsql/ral/common/set/executor/SetVariableExecutorTest.java [hunk 1]
```diff
@@ -40,7 +44,7 @@ public void assertExecuteWithTransactionType() {
         BackendConnection connection = mock(BackendConnection.class);
         when(connection.getTransactionStatus()).thenReturn(new TransactionStatus(TransactionType.XA));
         new SetVariableExecutor(statement, connection).execute();
-        Assert.assertThat(TransactionType.LOCAL.name(), is(connection.getTransactionStatus().getTransactionType().name()));
+        assertThat(TransactionType.LOCAL.name(), is(connection.getTransactionStatus().getTransactionType().name()));
```
Comment (TeslaCN) [inline shardingsphere-proxy/shardingsphere-proxy-backend/src/test/java/org/apache/shardingsphere/proxy/backend/text/distsql/ral/common/set/executor/SetVariableExecutorTest.java:47]: Actual value is on the left.

File: shardingsphere-proxy/shardingsphere-proxy-backend/src/test/java/org/apache/shardingsphere/proxy/backend/text/distsql/ral/common/set/executor/SetVariableExecutorTest.java [hunk 2]
```diff
@@ -49,15 +53,20 @@ public void assertExecuteWithAgent() {
         BackendConnection connection = mock(BackendConnection.class);
         new SetVariableExecutor(statement, connection).execute();
         String expectedValue = SystemPropertyUtil.getSystemProperty(VariableEnum.AGENT_PLUGINS_ENABLED.name(), "default");
-        Assert.assertThat(expectedValue, is("false"));
+        assertThat(expectedValue, is("false"));
```
Comment (TeslaCN) [inline shardingsphere-proxy/shardingsphere-proxy-backend/src/test/java/org/apache/shardingsphere/proxy/backend/text/distsql/ral/common/set/executor/SetVariableExecutorTest.java:56]: Maybe the `expectedValue` here and below should be named `actualValue`?

**Review by TeslaCN — CHANGES_REQUESTED** [review]

**Comment by TeslaCN** [comment]
You may find the code of conduct in our documents
https://shardingsphere.apache.org/community/en/contribute/code-conduct/

**Comment by lanchengx** [comment]
> You may find the code of conduct in our documents https://shardingsphere.apache.org/community/en/contribute/code-conduct/

Thanks a lot, I will pay more attention.

**Review by TeslaCN — APPROVED** [review]

**Pull request merged**